include/triton/Dialect/TritonLang/IR/TritonLangOps.td

```diff
@@ -0,0 +1,102 @@
+#ifndef TRITON_LANG_OPS
+#define TRITON_LANG_OPS
+
+include "mlir/Dialect/Arith/IR/ArithBase.td"
+include "triton/Dialect/Triton/IR/TritonTypes.td"
+include "triton/Dialect/TritonLang/IR/TritonLangDialect.td"
+
+//
+// Op Base
+//
+class TL_Op<string mnemonic, list<Trait> traits = []> :
+    Op<TritonLang_Dialect, mnemonic, traits> {
+}
+
+//
+// Make Range Op
+//
+// TODO: should have ConstantLike as Trait
+def TL_MakeRangeOp : TL_Op<"make_range", []> {
+    let summary = "make range";
+
+    let description = [{
+        Returns an 1D int32 tensor.
+
+        Values span from $start to $end (exclusive), with step = 1
+    }];
+
+    let arguments = (ins I32:$start, I32:$end);
```

Comment:
MakeRange op is cloned here since the `tt.make_range` can only accept Int32Attr, while we need to represent with dynamic value here.